src/pages/node/aliases.tsx

```
@@ -24,64 +23,44 @@ import { OpenSessionModal } from '../../components/Modal/node/OpenSessionModal';
-    if (!loginData.apiEndpoint) return;
+    if (!myNodeAddress) return;
     for (const data of parsedData) {
-      if (data.alias && data.peerId) {
-
-      }
+      dispatch(
+        nodeActions.setAlias({
+          nodeAddress: data.address || data.nodeAddress,
+          alias: data.alias,
+        })
+      )
```

ESLint

[error] Missing trailing comma.

(comma-dangle)